demos/coreHTTP/http_demo_s3_download_multithreaded.c

```diff
@@ -589,8 +583,8 @@ static BaseType_t prvDownloadS3ObjectFile( const char * pcHost,
                 ulWaitCounter = 0;
                 xRemainingResponseCount -= 1;
             }
-            /* Check for a notification from the HTTP task about an HTTP send failure. */
-            else if( prvCheckNotification( &ulNotification, httpexampleHTTP_SEND_ERROR, pdTRUE ) != pdFALSE )
+            /* The HTTP task has notified about an HTTP send failure. */
```

Comment:
nit: this comment can probably be deleted since the `LogError()` on line 589 says the same thing.

Reply:
Removed the comment